Jenkinsfile

```diff
@@ -9,9 +9,25 @@ pipeline {
             }
         }
 
+       stage("Yosys synthesis"){
+            steps {
+               sh 'docker run --rm -v $(pwd):/src -w /src librecores/librecores-ci:0.4.0 /bin/bash -c \
+               "fusesoc library add mor1kx /src; \
+                fusesoc run --target=synth mor1kx; \
+                /test-scripts/extract-yosys-stats.py < build/mor1kx_*/synth-icestorm/yosys.log"'
+            }
+        }
+
+        stage('Performance tests') {
```

Comment:
This is not really performance tests it's  Resource Usage Report right?  Can  the name be made to reflect that?

Reply:
Done